src/App.js

```diff
@@ -17,45 +20,117 @@ What should a cart item look like? 🤔
 console.log(initialStoreItems)
 
 export default function App() {
+ 
+  const [cart, setCart] = useState([])
   // Setup state here...
 
+  const addToCart = selection => {
+    // if selected item is not in the cart, the cart will 
+    // from quantity 1
+    if (!cart.includes(selection)) {
+      console.log("cart", !cart, "selection", selection)
+      selection.quantity = 1
```

Comment:
A very lean method of writing into the state. Does this satisfy the requirements of creating a new array though?